setup.py

```diff
@@ -347,7 +347,14 @@ def build_libs(libs):
 
     my_env["BUILD_TORCH"] = "ON"
 
-    if subprocess.call(build_libs_cmd + libs, env=my_env) != 0:
+    try:
+        os.mkdir('build')
+    except OSError:
+        pass
+
+    kwargs = {'cwd': 'build'} if not IS_WINDOWS else {}
```

Comment:
What's going on here?

Reply:
for the linux/mac script, i've made the output go to the current directory - so now we have to make the current directory be `build` for things to go in the right place.

it's some localized messiness for the moment, but it should get cleaned up in passing as we merge build_pytorch_libs.{sh,bat} into the underlying cmake layer

Reply:
It is always good when something is intended to be temporary to say so in a comment :)